modules/nlu/src/views/full/intents/index.jsx

```diff
@@ -1,39 +1,20 @@
 import React from 'react'
-import { Button, Glyphicon, ListGroup, ListGroupItem, FormGroup, InputGroup, FormControl } from 'react-bootstrap'
+import { Container } from 'botpress/ui'
 import _ from 'lodash'
 
 import IntentEditor from './editor'
-import style from '../style.scss'
+import NLUSidePanel from './SidePanel'
 
 export class IntentsComponent extends React.Component {
   state = {
     showNavIntents: true,
     intents: [],
-    currentIntent: null,
-    filterValue: ''
+    currentIntent: null
   }
 
   componentDidMount() {
     this.fetchIntents()
     this.fetchContexts()
-    this.checkForIntentSearch()
-  }
-
-  // Deprecated, will be fixed when we fix the whole NLU UI
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.intent !== this.props.intent) {
-      this.initiateStateFromProps(nextProps)
-    }
-  }
-
-  checkForIntentSearch() {
-    const { hash } = window.location
-    const searchCmd = '#search:'
-
-    if (hash && hash.includes(searchCmd)) {
-      const intent = hash.replace(searchCmd, '')
-      this.setState({ filterValue: intent, currentIntent: intent })
-    }
```

Comment:
Should we keep that ? It was used by the emulator so we could click on the intent to get here directly. Not sure it would work well with the channel web insite the iframe, but maybe we may want to keep it around